core/commands/ls.go

```diff
@@ -78,12 +78,13 @@ it contains, with the following format:
 				Links: make([]LsLink, len(dagnode.Links)),
 			}
 			for j, link := range dagnode.Links {
-				link.Node, err = link.GetNode(req.Context(), node.DAG)
+				var linkNode *merkledag.Node
+				linkNode, err = link.GetNode(req.Context(), node.DAG)
```

Comment:
I didn't want to override the `err` variable. using `:=` is not equivalent because it creates a new `err` variable scoped in the `for` loop.